File: Examples/DotNET/CSharp/Slides/GetSlideColorScheme.cs
using System;
using Com.Aspose.Slides.Api;
using Com.Aspose.Slides.Model;
using Com.Aspose.Storage.Api;

namespace Slides
{
    class GetSlideColorScheme
    {
        public static void Run()
        {
            // ExStart:1
            SlidesApi slidesApi = new SlidesApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "demo.pptx";
            int slideIndex = 1;
            String folder = "";
            String storage = "";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Slides Cloud SDK API to get slide color scheme
                ColorSchemeResponse apiResponse = slidesApi.GetSlidesThemeColorScheme(fileName, slideIndex, folder, storage);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    ColorScheme colorScheme = apiResponse.ColorScheme;
                    Console.WriteLine("Accent1 : " + colorScheme.Accent1);
                    Console.WriteLine("Accent3 : " + colorScheme.Accent2);
                    Console.WriteLine("Dark1 : " + colorScheme.Dark1);
                    Console.WriteLine("Dark2 : " + colorScheme.Dark2);
                    Console.WriteLine("Light1 : " + colorScheme.Light1);
                    Console.WriteLine("Get Color Scheme of a PowerPoint Slide, Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);

            }            
            // ExEnd:1
        }
    }
}

File: Examples/Node.js/Slides/GetSlideColorScheme.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var SlidesApi = require('asposeslidescloud');
var StorageApi = require('asposestoragecloud');
var configProps = require('../Data/config.json');
var data_path = '../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose.Slides API SDK
var slidesApi = new SlidesApi(config);

// Set input file name
var name =  "sample-input.pptx";
var slideIndex = 1;

var versionId = null;
var storage = null;
var folder = null;

try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, versionId, storage, data_path + name , function(responseMessage) {

	assert.equal(responseMessage.status, 'OK');

	// Invoke Aspose.Slides Cloud SDK API to get font scheme of a particular slide
	slidesApi.GetSlidesThemeColorScheme(name, slideIndex, folder, storage, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			var colorScheme = responseMessage.body.ColorScheme;
			console.log("Accent1 : " + colorScheme.Accent1);
			console.log("Accent3 : " + colorScheme.Accent2);
			console.log("Dark1 : " + colorScheme.Dark1);
			console.log("Dark2 : " + colorScheme.Dark2);
			console.log("Light1 : " + colorScheme.Light1);
			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1